streams.yml

```diff
@@ -11,6 +11,8 @@ elasticsearch:
   image: openshift/ose-base:elasticsearch
 ruby-25:
   image: rhscl/ruby-25-rhel7
```

Comment:
as logging-fluentd is the only consumer of ruby-25, the rhel7-based stream could technically be removed here.

Reply:
Thanks for pointing it out, fixed.